guide/multiple-wallets.md

```diff
@@ -0,0 +1,313 @@
+
+Certain aspects of the application need to be flexible. For example, when a user scans a [QR code]({{ '/guide/daily-spending-wallet/requesting/#qr-code' | relative_url }}) of an invoice, an appropriate wallet can be automatically chosen. But this might not match the user intent, so they should be able to easily switch.
+
+Let's take a closer look at some of the important screens and user flows.
+
+## Payments screen
+
+Compared to the [daily spending wallet]({{ '/guide/daily-spending-wallet/requesting/#requesting-entry-point' | relative_url }}), the only addition to the payments screen is an indicator for the currently selected wallet, right above the amount input. This indicator also functions as a button to expand the wallet selector modal.
+
+The modal lists all available wallets, along with their balances. In our scenario, the balance of the inheritance wallet is not visible, since the user has set up a [separate PIN]({{ '/guide/daily-spending-wallet/privacy/#entering-a-pin-to-reveal-information' | relative_url }}) for it. They are concerned about accidentally exposing this information to bystanders.
+
+{% include image-gallery.html pages = page.images_payments %}
+
+## Balance & activity
+
+By default, balance and [activity]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) only include information from the currently selected wallet. A wallet switcher is available again right above the balance. This switcher includes the option to select multiple (or all) wallets, so users can more easily get an overview of all their funds. This selection carries through to the search functionality.
+
+{% include image-gallery.html pages = page.images_activity %}
+
+## Sending
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/multiple-wallets/send.png"
+   retina = "/assets/images/guide/multiple-wallets/send@2x.png"
+   modalImage = "/assets/images/guide/multiple-wallets/send-big.png"
+   alt-text = "Send screen with a wallet selector option"
+   caption = "Users can choose which wallet to send from"
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+A wallet indicator and switcher in the [send flow]({{ '/guide/daily-spending-wallet/sending/' | relative_url }}) is a necessity to keep users informed about which funds will be transferred. Applications may automatically switch wallets to match given payment information. For example, if the user selects a contact that only has a lightning address, the application should switch to the lightning wallet.
+
+Users should be able to easily override these if they want to act differently based on their current needs and goals. For example, an application may decide that the on-chain wallet is most appropriate for a payment because the lightning wallet is low on liquidity. But the user might be in a rush and prefers to use lightning even if it means paying a swap fee for the additional liquidity.
```

Comment:
```suggestion
Users should be able to easily override these if they want to act differently based on their current needs and goals. For example, an application may decide that the on-chain wallet is most appropriate for a payment because the lightning wallet is low on liquidity. However the user might be in a rush and prefers to use lightning even if it means paying a swap fee for the additional liquidity.
```